randovania/games/planets_zebeth/exporter/patch_data_factory.py

```diff
@@ -106,6 +106,9 @@ def _create_game_config_dict(self):
             "seed_identifier": self._create_hash_dict(),
             "starting_items": self._create_starting_items_dict(),
             "starting_memo": self._create_starting_memo(),
+            "warp_to_start": False,
```

Comment:
if these only work in the new patcher version, then you should bump the patcher version in this PR

Reply:
Bump it yourself, cherry pick the commit from that branch, start your branch from that one. All it does is automatically make a pr so we know there's an update.

Reply:
Oh, Henrique closed the PR.
If the PR was still open I would've merged it. But it's nicer in a lot of cases to have patcher bump and pdf in the same pr so that it's easier to revert them